modules/metastorage/pom.xml

```diff
@@ -56,5 +56,29 @@
             <artifactId>ignite-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.ignite</groupId>
+            <artifactId>ignite-vault</artifactId>
+            <version>${project.version}</version>
```

Comment:
Please do not specify the version here. It seems that all concrete versions should be defined int the parent.pom.
https://issues.apache.org/jira/browse/IGNITE-14630